ddl/attributes_sql_test.go

```diff
@@ -50,7 +53,13 @@ func (s *testDBSuite8) TestAlterTablePartitionAttributes(c *C) {
 	tk := testkit.NewTestKit(c, s.store)
 	tk.MustExec("use test")
 	tk.MustExec("drop table if exists t1")
-	defer tk.MustExec("drop table if exists t1")
+	defer func() {
+		tk.MustExec(`alter table t1 partition p0 attributes="";`)
+		tk.MustExec(`alter table t1 partition p1 attributes="";`)
+		tk.MustExec(`alter table t1 partition p2 attributes="";`)
+		tk.MustExec(`alter table t1 partition p3 attributes="";`)
+		tk.MustExec("drop table if exists t1")
```

Comment:
drop table should drop attributes of partitions too, I think. We should not need to add `alter table t1 partition p3 attributes=""`, if we added, something is wrong.

Reply:
The reason why I do it is to prevent the influence among cases.